cstool/cstool.c

```diff
@@ -384,6 +389,10 @@ int main(int argc, char **argv)
 				if (cs_support(CS_ARCH_EVM)) {
 					printf("evm=1 ");
 				}
+				
+				if (cs_support(CS_ARCH_RISCV)) {
```

Comment:
Not here, but in function `usage` the support informations are missing for RISCV.